pyproject.toml

```diff
@@ -0,0 +1,79 @@
+[build-system]
+requires = ["setuptools>=40.9.0", "wheel", "setuptools_scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pip-licenses"
+version = "4.4.0"
```

Comment:
Great! Thank you.

When I run `make build` in my environment, it seems to have packaged it without any problems.

So we plan to upload the packages created in this branch to [Test PyPI](https://test.pypi.org/project/pip-licenses/).

I'm wondering what to do with the next version as semantic versioning, could you commit it as ......`4.6.0rc1` ?

Reply:
I think so...